service/resource/deployment/template.go

```diff
@@ -0,0 +1,18 @@
+package deployment
+
+import "fmt"
+
+const (
+	templateURIVersionDefault = "master"
+	templateURIFmt            = "https://raw.githubusercontent.com/giantswarm/azure-operator/%s/service/arm_templates/%s"
+
+	mainTemplate = "main.json"
```

Comment:
I think these should be exported and have godoc.

Reply:
It's used only inside the package. I can add a godoc but I have no idea what it should be.

`mainTemplate is the name of the main template`? It's ridiculous.

Reply:
It could explain what the main template is. I have no idea what name that is and why it is in JSON and so on.

Reply:
This is it: https://github.com/giantswarm/azure-operator/blob/master/service/arm_templates/main.json

Reply:
So this is the name of the main ARM template. I don't know how to explain that better. I can rename to `mainARMTemplate` if that makes sense. ARM stands for Azure Resource Manager. Those templates are executed with deployments. The thing is I don't want to create Azure documentation in the godoc.